tests/mr/device/mr_test.hpp

```diff
@@ -243,6 +244,11 @@ inline auto make_pool()
   return rmm::mr::make_owning_wrapper<rmm::mr::pool_memory_resource>(make_cuda());
 }
 
+inline auto make_arena()
+{
+  return rmm::mr::make_owning_wrapper<rmm::mr::arena_memory_resource>(make_pool());
```

Comment:
Is arena_mr intended to work with upstream resources other than pool?

Reply:
It now works directly with cuda_mr (or managed_mr).